spec/xcpretty/parser_spec.rb

```diff
@@ -198,6 +198,16 @@ module XCPretty
       @parser.parse(SAMPLE_MEASURING_TEST)
     end
 
+    it "parses build success indicator" do
+      @formatter.should receive(:format_phase_success).with('BUILD')
+      @parser.parse(SAMPLE_BUILD_SUCCEEDED)
+    end
+
+    it "parses clean success indicator" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.